core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -287,8 +298,8 @@ contract Liquidatable is PricelessPositionManager {
         // Enqueue a request with the DVM.
         _requestOraclePrice(disputedLiquidation.liquidationTime);
 
-        // Pay a final fee
-        _payFinalFees(msg.sender);
+        // Pay a final fee.
+        _payFinalFees(msg.sender, disputedLiquidation.finalFee);
```

Comment:
Do you think there is any use in having a `require(disputerBalance >= (disputeBond + finalFee)` check or is it sufficient to just have the `safeTransferFrom`'s fail if the disputer doesn't have enough balance?

Reply:
I figure it's just as good to let the transfer from fail since that will catch "balance too low" errors *and* "approval too low" errors. Do you see any advantage to having a separate require for the caller's balance?

Reply:
This is sufficient IMO.